src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.jpa.hibernate.ddl-auto=create
+spring.datasource.url=jdbc:mysql://localhost:3306/keyboard_builder
+spring.datasource.username=admin
+spring.datasource.password=testtest
```

Comment:
Local WAMP server requires a user created with the name admin, and testtest for this configuration to work properly

Reply:
I can use MySQL with admin and no password, by default.